x/stakeibc/keeper/icqcallbacks_callibrate_delegation.go

```diff
@@ -11,6 +11,9 @@ import (
 	"github.com/Stride-Labs/stride/v15/x/stakeibc/types"
 )
 
+// CalibrationThreshold is the max amount of tokens by which a calibration can alter internal record keeping of delegations
+var CalibrationThreshold = sdk.NewInt(1000)
```

Comment:
Injective actually has the potential to reach this pretty quick cause of the granular microdenom. We can either raise this or just commit to running calibration frequently. Wdyt?

Reply:
Raised to 100,000 (0.1 macrodenom for 1e-6 chains). Still may get hit fairly quickly on 1e-18 chains, but gives us more slack and we can do calibrations.

Reply:
This feels a bit more dangerous - I think I'd advocate for keeping it at 1000 for now and monitoring injective over the next couple weeks before we push this live and then re-evaluate.

We could also just remove this tx altogether and issue it epochly

Reply:
That makes sense. I'll leave it at 1000.

Reply:
Looks like you set it to 100,000 in a recent commit, so I think you need to drop it back down to 1000

Reply:
looking at injective, they got off by ~2200 over the course of probably 2 weeks. 

Let's do either like 3k or 10k

Reply:
Set to 3k

Reply:
I raised this to 5000 in my other PR - I think it'd be good to have a little more leeway